lib/clevertap_plugin_web.dart

```diff
@@ -5,13 +5,13 @@ import 'package:flutter/services.dart';
 import 'package:flutter_web_plugins/flutter_web_plugins.dart';
 import 'package:js/js_util.dart' as js_util;
 import 'package:js/js.dart';
-import 'dart:html' as html;
 
 /// A web implementation of the CleverTapPlugin plugin.
 class CleverTapPlugin {
   static MethodChannel? _nativeToDartMethodChannel;
 
   static void registerWith(Registrar registrar) {
+    print("registerWith");
```

Comment:
Was this only for internal testing purposes or does it need to be a part of the sdk?

Reply:
That was for internal testing

Reply:
Can be removed then?

Reply:
Done.